src/api/app/views/webui/requests/change_devel_projects/new.html.haml

```diff
@@ -0,0 +1,22 @@
+- @pagetitle = 'Change Devel Project Request'
+
+.row
+  .col
+    .card.mb-3
+      = render partial: 'webui/package/tabs', locals: { project: @project, package: @package }
+      .card-body
+        .row
+          .col-12.col-md-8.col-lg-6
+            %h5 Change Devel Project Request
+            = form_tag(create_packages_change_devel_projects_path(project: @project, package: @package), method: :post) do
+              %p
+                -# FIXME: get rid of this helper
+                Do you want to request to change the devel project for #{package_link(@package)} from
+                \#{project_or_package_link(project: @package.develpackage.project.name)}?
+              = render partial: 'webui/shared/autocomplete', locals: { html_id: 'devel_project',
+                                                                       label: 'New Devel Project:',
+                                                                       data: { source: autocomplete_projects_path } }
+              .form-group
+                = label_tag(:change_devel_description, 'Description:')
+                = text_area_tag(:change_devel_description, '', row: 3, class: 'form-control')
+              = submit_tag('Create', class: 'btn btn-primary px-4')
```

Comment:
I suggest using `Request` here since we're requesting to change the devel project.

Reply:
Done.